### PR TITLE
Remove wooden tools

### PR DESCRIPTION
**Pull request by ClobberXD**
Wooden tools only make sense in survival games like MTG, where they make up the first step in the survival ladder, as a bridge between bare hands and stone tools. In CTF, wooden tools can't be crafted, and don't noticeably deal more damage than bare hands anyway.

Tested. Works.

### CODE REVIEW

**Comment by ClobberXD** [comment]
With the maps submodule issue fixed, I could successfully test this PR. Couldn't find any problems.

**Comment by bewlay** [comment]
I like this. Tested, found no problems either.

**Comment by ClobberXD** [comment]
I feel wooden tools don't make much sense in "pure-PvP" environment such as CTF. Removing these tools allows #223 to space out the weapons even better.

**Comment by AKryukov92** [comment]
> I feel wooden tools don't make much sense in "pure-PvP" environment such as CTF.

Why? Wooden tools are great to be "default" weapons.
What do you want to give to players on respawn instead of wood?
Steel tools? But you devalue steel tools this way. They are more complex to craft compared to anything else. Player have to get limited steel and fuel, craft furnace, hide furnace, wait for smelting. Good stats of steel tools is a great reward for all these troubles.
Stone tools? They are some kind of reward too. Better stats for having trouble to open crafting interface and dig some stone.

> Removing these tools allows #223 to space out the weapons even better.

If this is an issue, then we should remove diamond instead. It is overpowered new and have similar reward idea as mese. In both cases player should know where to dig to get them. They don't even need to smelt anything.

**Comment by Thomas--S** [comment]
>>I feel wooden tools don't make much sense in "pure-PvP" environment such as CTF.

>Why? Wooden tools are great to be "default" weapons.
What do you want to give to players on respawn instead of wood?
Steel tools? But you devalue steel tools this way. They are more complex to craft compared to anything else. Player have to get limited steel and fuel, craft furnace, hide furnace, wait for smelting. Good stats of steel tools is a great reward for all these troubles.
Stone tools? They are some kind of reward too. Better stats for having trouble to open crafting interface and dig some stone.

:+1: 

---



>>Removing these tools allows #223 to space out the weapons even better.

>If this is an issue, then we should remove diamond instead. It is overpowered and have similar reward idea as mese. In both cases player should know where to dig to get them. They don't even need to smelt anything.

:-1:

**Comment by afshaan4** [comment]
>Why? Wooden tools are great to be "default" weapons.
What do you want to give to players on respawn instead of wood?
Steel tools? But you devalue steel tools this way. They are more complex to craft compared to anything else. Player have to get limited steel and fuel, craft furnace, hide furnace, wait for smelting. Good stats of steel tools is a great reward for all these troubles.
Stone tools? They are some kind of reward too. Better stats for having trouble to open crafting interface and dig some stone.


wood weapons are about as effective as hitting the enemy with feathers and so the contrast to a weapon that is `>=` steel is very high, meaning that they are *ABSOLUTELY* **useless**, you might as well spawn with nothing coz they use up precious inventory top bar space.

stone tools should be made the default weapon instead, they aren't as deadly as weapons that are `>=` steel but are effective enough that a mob of people with them can take out a single enemy with a better weapon

**Comment by LoneWolfHT** [comment]
I'm all for removing wooden tools and using stone as the defaults. Makes things simpler.

**Comment by afshaan4** [comment]
>Your words don't answer the question: why do you want to remove wood (and make stone default) rather than increase damage of wood?

because stone doesn't need tweaking to make it useful, and by removing woodq tools we make the weapons selection leaner (which I think will be beneficial)

also if we made wood tools deal more damage then they'd make stone tools redundant, and we'd be back at the same issue of having one unnecessary and unused tool category

**Comment by AKryukov92** [comment]
> by removing woodq tools we make the weapons selection leaner (which I think will be beneficial)

It is not valid argument. We could also make weapon selection leaner by removing overpowered diamond and mese tools. Which will be beneficial too.
Why do you want to remove something, which is good as default?

> They're outright useless in non-survival environments

Wood sword is useless in CTF only because it has low damage. If we tweak its damage and speed, then it will be useful.

I also don't like the fact that we just give steel picks by default on some maps. We are just removing "crafting" part of the game, which is important.

> I'm not sure why you want to tweak something that's better removed altogether.

I have explained my point in https://github.com/MT-CTF/capturetheflag/pull/331#issuecomment-456710266
Wooden tools are great to be "default" weapons.
Stone tools are some kind of reward too. Better stats for having trouble to dig some stone and open crafting interface.
Steel tools are more complex to craft compared to anything else. Player have to get limited steel and fuel, craft furnace, hide furnace, wait for smelting. Good stats of steel tools is a great reward for all these troubles.
Diamond and mese tools are overpowered because they provide high stats and don't need any complex crafting. You just need to know location of resources and dig there faster than anyone else. Swords out of these minerals could kill too fast to be able to react.

We should remove diamond and mese tools instead of wooden ones.

**Comment by afshaan4** [comment]
>I also don't like the fact that we just give steel picks by default on some maps. We are just removing "crafting" part of the game, which is important.


that's some of those maps need them, like the caverns which would be a pain to play on without a steel pick

>It is not valid argument

that's your opinion
and about making bronze the most powerful: no, I personally do not want to spend time going through a long crafting process for a sword when a raging battle is going on.

**Comment by AKryukov92** [comment]
> I find the mese swords quite fun to use

Of course it is fun to use overpowered weapon. But it is still overpowered.
One just need to know where to dig to begin spawnkilling.

**Comment by ClobberXD** [comment]
> One just need to know where to dig to begin spawnkilling.

Spawnkilling's sole existence is fuelled by players respawning with wooden tools. Giving them stone tools will instil a sense of fear in the spawnkiller, as the respawned players have a much higher chance of eliminating them. Players would still look up to, and work hard for steel tools, as they're still significantly better than stone tools.

**Comment by AKryukov92** [comment]
> Spawnkilling's sole existence is fuelled by players respawning with wooden tools

It is not because they spawn with **wood**. It is because wood has **low damage**. Do you feel the difference?
If you increase damage of wood tools, then you solve this problem too.

**Comment by ClobberXD** [comment]
> It is because wood has low damage

And there are two ways to deal with this:
- Tune the damage numbers
- Remove it altogether

I'm just proposing to remove it altogether, backed by the rationale in the first post.

**Comment by LoneWolfHT** [comment]
>It is not because they spawn with >wood. It is because wood has low >damage.

You just agreed and disagreed with them at the same time?

I NEVER go to craft a stone sword when there is no steel. I go mine steel to craft a steel sword. So removing wood and making stone the default is the way to do it IMO

**Comment by LoneWolfHT** [comment]
> You don't craft stone sword, but you want stone sword as default. Why exactly you don't craft it?

Because it's more beneficial to just go mine for a steel sword. The enemy team will almost always have steel weapons. And stone is pretty much useless against them.

**Comment by LoneWolfHT** [comment]
> A possible problem mentioned on the server is that wooden swords/picks are actually useful sometimes, because you can craft them to wood to make swords. I think at least the map where one or both teams don't have trees nearby (The Bridge, maybe Coast of Nostalgia), initial stuff should include sticks.

Or you could just change the crafts to use stone tools instead of wood ones. Maybe making them give the amount of sticks used instead of wood?

**Comment by BrianGaucher** [comment]
I agree that wood swords are nearly useless since they do little more than bare hands. But some wood tools are useful since you can't dig stone without a wood pickaxe.

My view is removing wood swords from chest, and removing the wood sword from crafting. Maybe keeping it around as a spawning weapon for map makers (unless we just start with fist instead). But I think wood picks should be kept since it's then possible to punch trees, get a wood pick, dig stone, make a stone pick.

TLDR; Remove wood swords from chest and crafting (maybe keep them for spawning weapons, or use fist), but keep wood picks. (And wood shovels ?)

**Comment by LoneWolfHT** [comment]
> I agree that wood swords are nearly useless since they do little more than bare hands. But some wood tools are useful since you can't dig stone without a wood pickaxe.
> 
> My view is removing wood swords from chest, and removing the wood sword from crafting. Maybe keeping it around as a spawning weapon for map makers (unless we just start with fist instead). But I think wood picks should be kept since it's then possible to punch trees, get a wood pick, dig stone, make a stone pick.
> 
> TLDR; Remove wood swords from chest and crafting (maybe keep them for spawning weapons, or use fist), but keep wood picks. (And wood shovels ?)

If we removed wooden tools instead of starting with wood players would start with stone. Unless they were a complete idiot they would never need to craft a wooden pick to mine stone

**Comment by rubenwardy** [comment]
Feel free to merge after merge rebase

**Comment by ClobberXD** [comment]
Rebased. But... you're the only one who's capable of merging :P
